### ISSUE TITLE
Creating a group recursively creates duplicate entries in LIBRARY_SEARCH_PATHS

### ISSUE THREAD

**Issue by mikeono** [kronenthaler/mod-pbxproj#188]
Run `add_folder('path/to/newfiles')` then inspect the project file changes in a text editor. You'll see that pbxproj created a new entry in `LIBRARY_SEARCH_PATHS` for each new file in the group, however it should only create one new entry since all the files are in the same directory.

```
project.add_folder('path/to/newfiles')
project.save()

```
It will look like this:

```
LIBRARY_SEARCH_PATHS = (
                                       "$(SRCROOT)/path/to/newfiles",
                                       "$(SRCROOT)/path/to/newfiles",
                                       "$(SRCROOT)/path/to/newfiles",
```

However it should look like this:

```
LIBRARY_SEARCH_PATHS = (
                                       "$(SRCROOT)/path/to/newfiles",
```

**Comment by kronenthaler** [comment]
Thanks for the report.

**Comment by kronenthaler** [comment]
@mikeono There was a ticket related to duplicated flags (fix release in v2.2.5) please confirm that also solves this issue.
